### ISSUE TITLE
Position and room setting dismissed after HomeBridge reboot

### ISSUE THREAD

**Issue by siavashvj** [wailuen/homebridge-sensibo-sky#11]
Every time my HomeBridge server restarts (Mac OS) the Sensibo items’ names, room settings and Favorites positions reset. 

This does not happen to other HomeBridge plugins.

**Comment by wailuen** [comment]
Hmm.. I am not able to replicate that. Let me try again this weekend.

**Comment by Cloudore** [comment]
This is a result of the Null Check implemented a couple of weeks ago, had the same issue.

**Comment by orpolaczek** [comment]
I can't replicate it. 
My homebridge is installed as a system service. 

Restarting / killing / rebooting the hosting machine still won't cause it and I have two pods in my account.

**Comment by siavashvj** [comment]
@Cloudore How did you solve it?

@orpolaczek I'm running HomeBridge as a system service as well (launchctl), and it happens every time.

**Comment by Cloudore** [comment]
@orpolaczek try restarting homebridge while having your Sensibo devices unplugged. In my country the electricity goes off and on like twice a day, so when that happens, Homebridge will boot before Sensibo, thus failing the Null check and completely removing all Sensibo devices from my Home.app, and then re-adding them as new when it discovers them.

@siavashj I created my fork and removed the Null Check by backtracking on @wailuen steps, since I'm no coder it took me 3 or 4 tries, but when I got it, I stopped having the issue.

**Comment by wailuen** [comment]
Allow me to remove the null check then.

**Comment by wailuen** [comment]
I found the part where it dislodge the registered device when NULL. Made a simple change but not able to test since it never happen to me. Ver 0.4.2. Please test and let me know if I can close the case

**Comment by siavashvj** [comment]
- Installed 0.4.2
- Rebooted HomeBridge (from Lauchctl)

It worked! Everything stayed put. 

Good job, and thank you!

**Comment by wailuen** [comment]
Perfect.
